lib/std/Build/CompileStep.zig

```diff
@@ -1414,6 +1416,9 @@ fn make(step: *Step) !void {
     if (self.link_gc_sections) |x| {
         try zig_args.append(if (x) "--gc-sections" else "--no-gc-sections");
     }
+    if (self.linker_dynamicbase) |x| {
+        if (!x) try zig_args.append("--no-dynamicbase");
```

Comment:
Is there any Windows-like target where dynamicbase is false by default?

Reply:
https://lists.gnu.org/archive/html/bug-binutils/2015-09/msg00204.html

Based on this, it looks like they have been defaulted to on since they were added in 2008.

Reply:
In that case, I guess we could leave the flag to default to `linker_dynamicbase: bool = true` since we don't distinguish between it being not overridden by the user nor set to true, would you say this is accurate? Or did I misunderstand it?

Reply:
That is correct, and that would make this logic more clear. Fixed.